index.js

```diff
@@ -0,0 +1,144 @@
+"use strict";
+
+const startPage = document.querySelector(".page-start"),
+    startBtn = startPage.querySelector(".startBtn-start");
```

Comment:
보통 변수는 한줄씩 각각 선언하는 경우가 많습니다. 그리고 가능하면 약어가 아닌 풀네임으로 작성해주세요. 이후 동일한 코멘트는 생략하겠습니다.
```suggestion
const startPage = document.querySelector(".page-start");
const startButton = startPage.querySelector(".startBtn-start");
```